continuous_integration/environment-mindeps.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
   # Distributed depends on the latest version of Dask
   - pip
   - pip:
-    - git+https://github.com/dask/dask
+    - git+https://github.com/hendrikmakait/dask@p2p-pandas
```

Comment:
Reminder to revert https://github.com/dask/distributed/pull/8635/commits/814c73ad16a91a76c7b59b12d1752e5c0c6f1645